ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
but if you look in the code below, they're immediately converted to string, so there's really no typing going on here at all. It won't work that way until `SetTag` takes an `interface{}` key.